modules/apps/29-fee/types/errors.go

```diff
@@ -14,4 +14,5 @@ var (
 	ErrCounterpartyAddressEmpty      = sdkerrors.Register(ModuleName, 7, "counterparty address must not be empty")
 	ErrForwardRelayerAddressNotFound = sdkerrors.Register(ModuleName, 8, "forward relayer address not found")
 	ErrFeeNotEnabled                 = sdkerrors.Register(ModuleName, 9, "fee module is not enabled for this channel. If this error occurs after channel setup, fee module may not be enabled")
+	ErrInvalidRefundAddress          = sdkerrors.Register(ModuleName, 10, "invalid refund address")
```

Comment:
not sure we will need this error once we allow multiple addresses to incentivize a packet